client/src/components/atoms/Tag.tsx

```diff
@@ -1,13 +1,15 @@
 import React, { FunctionComponent } from "react";
 
-interface Props {
-    text: string
+interface TagProps {
+  text: string,
+  dropdownItem?: boolean,
```

Comment:
Instead of adding more to Tag, add the styling you need to the user of Tag (i.e. SearchableDropdown.scss would have

.dropdown-item {
   .tag {
       padding: 6.65px 12.5px;
       background-color: $dropdown-tag-background;
    }
}